test/base/questionPages/CheckboxControllerBaseSpec.scala

```diff
@@ -60,7 +60,8 @@ abstract case class CheckboxControllerBaseSpec[A: Writes](controller: String, pa
   private def getRequest  = FakeRequest(GET, onPageLoadRoute)
   private def postRequest = FakeRequest(POST, onSubmitRoute).withFormUrlEncodedBody(("value", answer.toString))
 
-  def stubServiceSuccessfulSubmission = SelfEmploymentServiceStub(submitAnswerAndRedirectResult = Redirect(onwardRoute))
+  def stubServiceSuccessfulSubmission: SelfEmploymentServiceStub =
+    SelfEmploymentServiceStub(submitAnswerAndRedirectResult = Redirect(onwardRoute), hasOtherIncomeSources = Right(true))
```

Comment:
`hasOtherIncomeSources = Right(true)` is something you should override the stub in the ControllerSpec rather than the BaseSpec since it is only relevant to this one Controller, not all CheckboxControllers